webhooks-extension/docs/GettingStarted.md

```diff
@@ -0,0 +1,161 @@
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   0/2   ContainerCreating   0     0s
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   0/2   Running   0     6s
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   1/2   Running   0     6s
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   2/2   Running   0     6s
+```
+
+The 'sink' will then create the expected PipelineRun:
+
+```
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   Pending   0     1s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   Pending   0     1s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   Init:0/3   0     1s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   Init:1/3   0     4s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   Init:2/3   0     9s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   PodInitializing   0     10s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   5/6   Running   0     37s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   3/6   Running   0     2m38s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   2/6   Running   0     2m56s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   Completed   0     2m57s
+```
+
+Finally the 'sink' and '(webhook) Name' pods will be shut down by Knative:
+```
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   2/2   Terminating   0     98s
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   1/2   Terminating   0     100s
+buildah-hook-xwxm5-rp6s6-deployment-6674f7fdbb-rnwzz   2/2   Terminating   0     102s
+buildah-hook-xwxm5-rp6s6-deployment-6674f7fdbb-rnwzz   1/2   Terminating   0     105s
+buildah-hook-xwxm5-rp6s6-deployment-6674f7fdbb-rnwzz   0/2   Terminating   0     2m3s
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   0/2   Terminating   0     2m
+buildah-hook-xwxm5-rp6s6-deployment-6674f7fdbb-rnwzz   0/2   Terminating   0     2m8s
+buildah-hook-xwxm5-rp6s6-deployment-6674f7fdbb-rnwzz   0/2   Terminating   0     2m8s
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   0/2   Terminating   0     2m4s
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   0/2   Terminating   0     2m4s
+```
+
+You can use `kubectl logs [pod-name] --all-containers` to check the output of each pod in turn, and of course the Tekton dashboard for the pods managed by a PipelineRun. In the case of any problems, check that all of the above steps were correctly performed:
+- Create a service account and RoleBinding for the PipelineRuns to use
+- Create the correct Git and Docker credentials and patch the right service account
+- Ensure that your GitHub can route correctly to the webhooks-extension-sink: use `kubectl get ksvc` to check its value, and the GitHub web pages to see that the webhook was correctly created, and that it successfully delivered its result to the ksvc. 
```

Comment:
Worth mentioning `ksvc` == `kservice` and referencing the Knative docs for what one is?

Reply:
done